gbbs/benchmarks/BatchDynamicConnectivity/SkipList/SkipList.h

```diff
@@ -96,7 +99,8 @@ struct SkipList {
     };
 
     size_t n;
-    parlay::random rng = parlay::random(time(0));
+    //parlay::random rng = parlay::random(time(0));
+    parlay::random rng = parlay::random(4);//variable-arbitary seed 
```

Comment:
```suggestion
    auto const arbitrary_but_fixed_seed = 4u; 
    parlay::random rng = parlay::random( arbitrary_but_fixed_seed ); 
```
If your comments describe "what" then it is a good indication you can improve code readability with better identifiers. In this case, replacing the "magic constant" with a variable whose name captures exactly your comment achieves that.